jcp/parser.js

```diff
@@ -92,6 +90,31 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
     // http://www.psychiatrist.com:80/pcc/pages/categories.aspx?cat=Sleep
     result.rtype    = 'SEARCH';
     result.mime     = 'HTML';
+  } else if (/^\/advanced-search/i.test(path)) {
+    // https://www.psychiatrist.com/advanced-search/?title=insomnia
+    result.rtype    = 'SEARCH';
+    result.mime     = 'HTML';
+    result.unitid = parsedUrl.search.replace('?', '');
```

Comment:
Are you sure you want a ``unitid` with the value ``title=insomnia`` in this case ? Wouldn't it be better to have ``insomina`` in ``unitid`` ?

If this is the case, then it would be ``result.unitid = parsedUrl.query.title;``

If this solution works for you, consider updating the ``HTML SEARCH`` [analysis on Analogist](http://analyses.ezpaarse.org/platforms/5a87341f34a6dcf1b9b3183f/analyses/6157495224d4232f68f8800f)

Reply:
there should be no unitid in a SEARH since the unitid is not used for a SEARCH, its description is here : [https://ezpaarse-project.github.io/ezpaarse/essential/ec-attributes.html#unitid](https://ezpaarse-project.github.io/ezpaarse/essential/ec-attributes.html#unitid)

Reply:
You can make a custom field, for example search_fields (``result.search_fields``) and in ezPAARSE use the ``Output-Fields`` (https://ezpaarse-project.github.io/ezpaarse/features/outputfields.html) and ask to have the ``search_filters`` field in output